administrator/components/com_config/tmpl/component/default.php

```diff
@@ -42,10 +42,9 @@
 
 		<?php // Begin Sidebar ?>
 		<div class="col-md-2" id="sidebar">
-            <button class="navbar-toggler options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
-                 <span class="burger-toggler-icon">
-                     <?php echo Text::_('TPL_ATUM_TOGGLE_SIDEBAR'); ?>
-                  </span>
+            <button class="navbar-toggler btn-secondary my-2 options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
```suggestion
            <button class="navbar-toggler btn-secondary my-2 options-menu d-md-none d-lg-none d-xl-non" type="button" data-toggle="collapse" data-target=".sidebar-nav" aria-controls="sidebar-nav" aria-expanded="false" aria-label="Text::_('TPL_ATUM_TOGGLE_SIDEBAR'); ?>">
```

Dont hardcode to english